README.md

```diff
@@ -1,2 +1,15 @@
 # gem_of
```

Comment:
Not seeing anything here that tells me what this gets me.  Or how to use it once I have it..

Reply:
good call.  i can take a stab at that.  

how about:

Gemfiles are typically used solely for gems used for development and testing.  Are you tired of copypastaing the same Gems and rake tasks around to various projects, only to have to update them all later?
use the gem of gems!
we bring you:
rake, rototiller, rspec, rubocop, simplecov, yardstick, markdown, flay, flog, roodi, rubycritic, and coveralls for all your dev and unit testing needs.
we bring you:  beaker for your system testing needs, AND we ensure the bundle will install consistently across ruby versions by pinning old dependencies that don't seem to like following semver rules.

later we'll supply a bunch of canned rake tasks to bring this all together from the CLI or CI (see our Rakefile)

Reply:
That's good stuff.  As far as using... Is it just as simple as install it and then put that line in your gem file and you are good to go?

Reply:
That is definitely good information to add! Would it be helpful to include project URLs and a brief description of each gem:

[rake](https://github.com/ruby/rake): "A make-like build utility for Ruby."
[rototiller](https://github.com/puppetlabs/rototiller): "A Rake helper library for command-oriented tasks."
[rspec](http://rspec.info/): "Behaviour Driven Development for Ruby."
[rubocop](https://github.com/bbatsov/rubocop): "A Ruby static code analyzer, based on the community Ruby style guide."
[simplecov](https://github.com/colszowka/simplecov): "Code coverage for Ruby 1.9+ with a powerful configuration library and automatic merging of coverage across test suites."
[yardstick](https://github.com/dkubb/yardstick): "A tool for verifying YARD documentation coverage."
[markdown](https://en.wikipedia.org/wiki/Markdown): "A lightweight markup language with plain text formatting syntax."
[flay](https://github.com/seattlerb/flay): "Flay analyzes code for structural similarities."
[flog](https://github.com/seattlerb/flog): "Flog reports the most tortured code in an easy to read pain report."
[roodi](https://github.com/roodi/roodi): "Ruby Object Oriented Design Inferometer"
[rubycritic](https://github.com/whitesmith/rubycritic): "A Ruby code quality reporter."
[coveralls](https://docs.coveralls.io/): "Coveralls is a web service to help you track your code coverage over time, and ensure that all your new code is fully covered."

Also, I tried installing by cloning the project and doing `gem install --local gem_of` but got `ERROR:  Could not find a valid gem 'gem_of' (>= 0) in any repository`. I realized I needed to do `gem build gem_of.gemspec` first. Not sure if we want to mention this.

After installing I was able to add `gem_of` to a new project and got all of the dependencies with `bundle install --path vendor/bundle`. Awesome!

Reply:
yep